mlflow/pyfunc/__init__.py

```diff
@@ -415,6 +415,46 @@ def predict(self, data: PyFuncInput) -> PyFuncOutput:
     def unwrap_python_model(self):
         """
         Unwrap the underlying Python model object.
+
+        This method is useful for accessing custom model functions, while still being able to
+        leverage the MLflow designed workflow through the `predict()` method.
+
+        .. code-block:: python
```

Comment:
```suggestion
        .. test-code-block:: python
```

Can we add `test-`? We merged https://github.com/mlflow/mlflow/pull/7450 today to test code examples.